lib/base/strings.dart

```diff
@@ -197,7 +197,8 @@ abstract class Strings {
   static const change = 'Change';
   static const logOut = 'Log out';
   static const deleteAccount = 'Delete account';
-  static const showOnLeaderboard = 'Show me on the leaderboard';
+  static const appearAnonymous = 'Appear anonymous in leaderboard';
```

Comment:
should be 'on' leaderboard

Reply:
fixed in new commit, also changed it in the settings group description